web/src/user_group_edit.js

```diff
@@ -64,31 +64,38 @@ export function get_edit_container(group) {
     );
 }
 
-function update_add_members_elements(group) {
+function update_group_settings_add_members_elements(group) {
     if (!is_editing_group(group.id)) {
```

Comment:
Should we call this `update_group_settings_and_add_members_elements`?

Reply:
sure, updated!